### ISSUE TITLE
Update sector health rate to account for all sectors (not just active sectors)

### ISSUE THREAD

**Issue by BigLep** [davidgasquez/filecoin-storage-providers-market#15]
I believe the "Sector Health Rate" metric is currently

$$\frac{\text{Number of Active Sectors - Number of Faulted Sectors}}{\text{Number of Active Sectors}}$$

Where `Active Sectors` is sectors that are truly in the "Active State".  I think actually what we want is:

$$\frac{\text{Number of All Sectors - Number of Faulted Sectors}}{\text{Number of All Sectors}}$$

I think that will prevent the dips we see in the "Active Sectors" graph:

<img width="1819" alt="image" src="https://github.com/user-attachments/assets/3fa792b1-9e57-4b46-9b8d-ef257b7608d9">

https://filecoin-project.github.io/filecoin-storage-providers-market/provider/f01083914#durability

Basically I think we want to compare faulted sectors with the total number of sectors that an SP has, regardless of what state the sectors are in (looking at the states in https://lotus.filecoin.io/reference/lotus/state/#stateminerpartitions ).

**Comment by davidgasquez** [comment]
Heya @BigLep!

The [current definition](https://github.com/davidgasquez/filecoin-data-portal/blob/6f46a7f6afcef38b4e4b9de1e504c663e97b3358/dbt/models/filecoin_daily_storage_providers_metrics.sql#L374) is:

$$\frac{\text{Number of Active Sectors}}{\text{Number of Active Sectors + Number of Faulted Sectors}}$$

We are deriving [active sectors from the SP raw power](https://github.com/davidgasquez/filecoin-data-portal/blob/18c48d4261f123827c73c248ce5e7b1e85727477/dbt/models/filecoin_daily_storage_providers_metrics.sql#L35). With this definition, we're counting Active + Faulted sectors in each day calculations.

We have [daily sector events](https://github.com/davidgasquez/filecoin-data-portal/blob/6f46a7f6afcef38b4e4b9de1e504c663e97b3358/dbt/models/filecoin_daily_storage_providers_metrics.sql#L406-L412) for providers we could play with and add it to create an `AllSectors` but I wonder if we want to include things like `snapped` / `terminated`. 

A question that just popped up is: do `recovered` / `recovering` sectors count as Active? It not, we should change the formula to:

$$\frac{\text{Number of Active Sectors + Number of Recovering Sectors}}{\text{Number of Active Sectors + Number of Faulted Sectors + Number of Recovering}}$$

**Comment by BigLep** [comment]
Thanks @davidgasquez for the info.  I probably don't have mastery of all the sector states to speak with authority here.  

I think my reaction is stemming from the "right hand side" "active sectors" graph.  I think we actually want to graph "non terminated sectors".  Basically, I don't think the right hand side graph side should dip just because some sectors faulted.  I think that graph should represent the denominator of our metric (since I think we always want a denominator or "n count" when showing percentages).  

On the topic of `recovered` and `recovering` I don't think it hurts to add those especially if it enables the right hand side "denominator" graph to be more flat and not have drops.  

Does that make sense?
